### PR TITLE
New configure town zone multipliers

### PR DESCRIPTION
**Pull request by reldred**
Additional changes to the patch as discussed, now a straight calculation if the feature is enabled or not. Further clarification to menu configuration options. Alas I couldn't figure out how to hide the zone options unless the feature is set to 'True'.

todo when I learn how:
Create various presets
Make the settings menu items for the zone parameters only appear when the drop down is set to 'Custom' rather than 'Default' or one of the pre-sets.
Savegame stuff.

### CODE REVIEW

File: src/lang/english.txt [hunk 1]
```diff
@@ -1882,6 +1882,18 @@ STR_CONFIG_SETTING_ZOOM_LVL_OUT_16X                             :16x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_32X                             :32x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_64X                             :64x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_128X                            :128x
+STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculatiopns: {STRING2}
```
Comment (James103) [inline src/lang/english.txt:1885]:
```suggestion
STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculations: {STRING2}
```

File: src/lang/english_AU.txt [hunk 2]
```diff
@@ -1552,6 +1552,18 @@ STR_CONFIG_SETTING_ZOOM_LVL_NORMAL                              :Normal
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_2X                              :2x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_4X                              :4x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_8X                              :8x
+STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculatiopns: {STRING2}
```
Comment (James103) [inline src/lang/english_AU.txt:1555]:
```suggestion
STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculations: {STRING2}
```

File: src/lang/english_US.txt [hunk 3]
```diff
@@ -1656,6 +1656,18 @@ STR_CONFIG_SETTING_ZOOM_LVL_OUT_4X                              :4x
 STR_CONFIG_SETTING_ZOOM_LVL_OUT_8X                              :8x
 STR_CONFIG_SETTING_TOWN_GROWTH                                  :Town growth speed: {STRING}
 STR_CONFIG_SETTING_TOWN_GROWTH_HELPTEXT                         :Speed of town growth
+STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculatiopns: {STRING2}
```
Comment (James103) [inline src/lang/english_US.txt:1659]:
```suggestion
STR_CONFIG_SETTING_TOWN_ZONE_CALC_MODE                          :Use custom Town Zone calculations: {STRING2}
```

**Review by James103 — CHANGES_REQUESTED** [review]
I found an obvious typo, where "Use custom town zone calculatio**pn**s" should be "Use custom town zone calculatio**n**s".

**Comment by reldred** [comment]
> 
> 
> I found an obvious typo, where "Use custom town zone calculatio**pn**s" should be "Use custom town zone calculatio**n**s".

Thanks, I've committed those corrections, thanks!

**Comment by JGRennison** [comment]
Thank for the contribution :).
I've squashed and applied this with some minor adjustments.

**Pull request closed without merging**